classes/ConfigurationTest.php

```diff
@@ -44,7 +43,8 @@ class ConfigurationTestCore
         '/translations/export/index.php',
         '/webservice/dispatcher.php',
         '/upload/index.php',
-        '/index.php'
+        '/index.php',
+        '/vendor/autoload.php',
```

Comment:
I make the assumption here that it's better to invoke Composer's autoloader in case we will move more libraries in the future.